charts/sysdig/templates/configmap-benchmark-runner.yaml

```diff
@@ -1,14 +1,12 @@
-{{- if .Values.nodeAnalyzer.deploy }}
+{{- if and (not (include "deploy-nia" .)) .Values.nodeAnalyzer.deploy}}
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ template "sysdig.fullname" . }}-benchmark-runner
   labels:
 {{ include "sysdig.labels" . | indent 4 }}
 data:
-  {{- if .Values.nodeAnalyzer.collectorEndpoint }}
-  collector_endpoint: https://{{ .Values.nodeAnalyzer.collectorEndpoint }}
-  {{- end }}
+  collector_endpoint: https://{{ .Values.nodeAnalyzer.apiEndpoint | default .Values.nodeAnalyzer.collectorEndpoint }}
```

Comment:
The `if` block is not required anymore, as now the apiEndpoint is required so it will fail if not specified (see the image-analyzer configmap)